### ISSUE TITLE
[Question] Reproducing StarVLA-GR00T on RoboTwin 2.0 gives much lower success than the paper

### ISSUE THREAD

**Issue by Shark-Overlord** [starVLA/starVLA#322]
Hi, thanks for releasing the code and configs.

I am trying to reproduce StarVLA-GR00T on RoboTwin 2.0 with the multi-task unified policy setting (robotwin_all_50), but my result is much lower than the paper.

In the paper, Table 7 reports:

StarVLA-GR00T: 88.0 Clean / 88.5 Random
However, in my earlier run, the best clean result I observed was much lower:

robotwin_all_50
steps_150000
official RoboTwin evaluation
Clean average success: about 66.5%
I would like to ask whether I am still missing some important training or evaluation details.
### My training setup
**Current configuration:**
- Method: QwenGR00T
- Backbone: Qwen3-VL-4B-Instruct
- DINO backbone: dinov2_vits14
- Action model: DiT-B
- Action horizon: 50
- Dataset root: /media/datasets/RoboTwin_All
- Data mix: robotwin_all_50
- Seed: 42
- Max train steps: 150000
- Save interval: 10000
- Eval interval: 10000
- VLM per-device batch size: 4
- VLA per-device batch size: 16
- Gradient accumulation steps: 6
- Learning rate:
- qwen_vl_interface = 1e-5
- action_model = 1e-4
### **Current 8-GPU rerun**
I am now rerunning with 8 GPUs.

From the training log:

Num processes: 8
train_micro_batch_size_per_gpu = 16
gradient_accumulation_steps = 6
train_batch_size = 768
So the effective VLA global batch is:

8 * 16 * 6 = 768
And the VLM branch is:

8 * 4 * 6 = 192
I understand from the paper that the original setup used 48 A100 GPUs, and I am trying to align the effective batch size with fewer GPUs via gradient accumulation.

**Comment by Hong-King** [comment]
Perhaps your 'gradient accumulation' is set incorrectly, please check #86 . If it works, please tell me.

**Comment by Shark-Overlord** [comment]
{
    "fp16": {
        "enabled": false
    },
    "bf16": {
        "enabled": true
    },
    "train_micro_batch_size_per_gpu": "auto",
    "train_batch_size": "auto",
    "gradient_accumulation_steps": 6,
    "zero_optimization": {
        "stage": 2,
        "allgather_partitions": true,
        "allgather_bucket_size": 5e8,
        "reduce_scatter": true,
        "reduce_bucket_size": 5e8,
        "overlap_comm": true,
        "contiguous_gradients": true,
        "cpu_offload": false
    },
    "gradient_clipping": 1.0,
    "steps_per_print": 10
}

**Comment by JinhuiYE** [comment]
We will check the latest release on Robotwin

**Comment by JinhuiYE** [comment]
State consistency (most critical): Check whether you used state (joint angles, end-effector pose) during training but not during evaluation. This is a common killer — model expects it, eval doesn't provide it, performance collapses.

**Comment by tqace** [comment]
I’m experiencing the same issue as well.

**Comment by JinhuiYE** [comment]
> { "fp16": { "enabled": false }, "bf16": { "enabled": true }, "train_micro_batch_size_per_gpu": "auto", "train_batch_size": "auto", "gradient_accumulation_steps": 6, "zero_optimization": { "stage": 2, "allgather_partitions": true, "allgather_bucket_size": 5e8, "reduce_scatter": true, "reduce_bucket_size": 5e8, "overlap_comm": true, "contiguous_gradients": true, "cpu_offload": false }, "gradient_clipping": 1.0, "steps_per_print": 10 }

check this
